app/src/main/kotlin/com/wire/android/core/extension/Activity.kt

```diff
@@ -0,0 +1,13 @@
+package com.wire.android.core.extension
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+
+fun AppCompatActivity.replaceFragment(frameId: Int, fragment: Fragment, addToBackStack: Boolean = true) =
+    supportFragmentManager.doTransaction {
+        replace(frameId, fragment).apply {
```

Comment:
In terms of Kotlin Scope functions I think we should use ```also``` here. ```apply``` should be used when you are accessing properties and static fields of the object passed in the lambda.